### DISCUSSION TITLE
Organising files but not importing

### DISCUSSION THREAD

**Discussion by j-peeters** [beetbox/beets#4128] [Q&A] [answered]
I searched the docs but couldn't find an answer to the following:

Is it possible to have Beets organize a folder but not import the files into the database?
My use case would be that I have a large collection of singletons for DJing that I don't necessarily need to have in the database of my album collection.  

Thanks, Jan

**Comment by wisp3rwind** [comment, marked as answer]
I don't think so, that's not really the workflow that beets is build around. You could have a separate configuration file and database for these files, and use the [`BEETSDIR`](https://beets.readthedocs.io/en/stable/reference/config.html#environment-variable) environment variable to use that when importing these files:

```
BEETSDIR=/path/to/singleton/config/dir beets import <files>
```

**Reply by j-peeters** [reply]
That could be an idea. I'll research it. Thanks!
